zcash_client_backend/src/wallet.rs

```diff
@@ -432,8 +432,7 @@ pub type WalletOrchardOutput<AccountId> =
 
 /// An enumeration of supported shielded note types for use in [`ReceivedNote`]
 #[serde_with::serde_as]
-#[derive(Debug, Clone, PartialEq, Eq)]
-#[derive(serde::Serialize)]
+#[derive(Debug, Clone, PartialEq, Eq, serde::Serialize)]
```

Comment:
The serde derives were intentionally split to another line to make it easier to search them and put them behind a feature flag later. Not sure if this change was intentional or due to `fmt`

Reply:
i think it was `fmt`